mne/beamformer/tests/test_lcmv.py

```diff
@@ -211,6 +213,14 @@ def test_lcmv():
                   data_cov, reg=0.01, pick_ori="max-power", weight_norm=None,
                   max_ori_out='signed')
 
+    # Test if wrong channel selection is detected in application of filter
+    evoked_ch = deepcopy(evoked)
+    evoked_ch.pick_channels(evoked_ch.ch_names[:-1])
+    sfilter = make_lcmv_filter(evoked.info, forward_vol, data_cov, reg=0.01,
```

Comment:
@britta-wstnr don't be creative with variable names.
sometimes you can it sfilter or spat_filt etc. I vote you call it everywhere filters
so

filters = make_lcmv_filters(...)
stc = apply_lcmv(filters, ....
and then
stc = apply_lcmv_epochs(filters, ....
stc = apply_lcmv_raw(filters, ....

I think using operator is confusion as this would suggest that you can use an LCMV operator with apply_inverse that takes an "operator" as input